### DISCUSSION TITLE
Open /build/models: no such file or directory

### DISCUSSION THREAD

**Discussion by jblozano17** [mudler/LocalAI#537] [Q&A]
Following the instructions [here](https://github.com/go-skynet/LocalAI#example-use-gpt4all-j-model) I'm geting "open /build/models: no such file or directory" when executing curl http://localhost:8080/v1/models. 

Either the instructions are missing steps or there is something I'm not doing right (or both). Can anyone help?

Thanks!

**Comment by rai62** [comment]
Is there `ggml-gpt4all-j` bin in `models` directory?
